content/docs/setup/kubernetes/multicluster-install.md

```diff
@@ -91,18 +131,24 @@ plane. Creating secrets will register the secrets with Istio properly.
 
 > The local cluster running the Istio control plane does not need
 it's secrets stored and labeled. The local node is always aware of
-it's Kubernetes credentials, but the local node is not aware of
+its Kubernetes credentials, but the local node is not aware of
 the remote nodes' credentials.
 
 Create a secret and label it properly for each remote cluster:
 
 ```command
-$ pushd $HOME/multicluster
+$ pushd $WORK_DIR
 $ kubectl create secret generic ${CLUSTER_NAME} --from-file ${CLUSTER_NAME} -n istio-system
 $ kubectl label secret ${CLUSTER_NAME} istio/multiCluster=true -n istio-system
 $ popd
 ```
 
+{{< warning_icon >}}
+The secret name and the corresponding file name need to be the same.  Kubernetes secret
+data keys have to conform to `DNS-1123 subdomain` format, so the filename can't have
```

Comment:
I'd like a link to `DNS-1123 subdomain`

Reply:
I can add a link to RFC1123 but I don't want to go nuts trying to explain this problem.  The gist is that the folks who are generally aware of DNS format restrictions can understand this warning, e.g. that underscores are generally no good.  

The RFC doesn't specifically refer to "subdomain format" so is not straight forward to extract the relevant info from the RFC--e.g. nice use of MUST & SHOULD wording in https://tools.ietf.org/html/rfc1123#page-13 & https://tools.ietf.org/html/rfc952.  In this warning the restriction is I'm referring to is at the kubernetes level so it'd be more appropriate to link to k8s secret documentation.  However, this restriction on secret data key values is not explained at all in k8s documentation so the "DNS-1123 subdomain" info is from the error message thrown when the format check fails.